### PR TITLE
[bitnami/harbor] updated Harbor README parameters to match actual default value

### PR DESCRIPTION
**Pull request by walker-tom**
**Description of the change**

I noticed that the default value for `core.secretKey` as a random string whereas the Parameters Table said it was `not-a-secure-key`. I also added a default for `jobservice.secret` as currently it is missing from the table. 

The comment above `secretKey` in [values.yaml](https://github.com/bitnami/charts/blob/eeb7c8b0578efb7429ed4f37909576ca42f18e3a/bitnami/harbor/values.yaml#L914) and in [values-production.yaml](https://github.com/bitnami/charts/blob/eeb7c8b0578efb7429ed4f37909576ca42f18e3a/bitnami/harbor/values-production.yaml#L915) gives `not-a-secure-key` as an example, are you happy for this to remain or would you like me to remove it? 

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

### CODE REVIEW

File: bitnami/harbor/README.md [hunk 1]
```diff
@@ -278,7 +278,7 @@ The following tables list the configurable parameters of the Harbor chart and th
...
 | `coreImage.debug`           | Specify if debug logs should be enabled                                                                                                                                                                                                                                                  | `false`                                                 |
-| `core.secretKey`            | The key used for encryption. Must be a string of 16 chars                                                                                                                                                                                                                                | `not-a-secure-key`                                      |
+| `core.secretKey`            | The key used for encryption. Must be a string of 16 chars                                                                                                                                                                                                                                | _random 16 character long alphanumeric string_          |
```
Comment (miguelaeh) [inline bitnami/harbor/README.md:281]:
```suggestion
| `core.secretKey`            | The key used for encryption. Must be a string of 16 chars                                                                                                                                                                                                                                | Random 16 character long alphanumeric string            |
```

File: bitnami/harbor/README.md [hunk 2]
```diff
@@ -327,7 +327,7 @@ The following tables list the configurable parameters of the Harbor chart and th
...
 | `jobservice.affinity`             | Node/Pod affinities                                                                                                                                          | `{}` (The value is evaluated as a template)             |
 | `jobservice.podAnnotations`       | Annotations to add to the jobservice pod                                                                                                                     | `{}`                                                    |
-| `jobservice.secret`               | Secret used when the job service communicates with other components. If a secret key is not specified, Helm will generate one. Must be a string of 16 chars. |                                                         |
+| `jobservice.secret`               | Secret used when the job service communicates with other components. If a secret key is not specified, Helm will generate one. Must be a string of 16 chars. | _random 16 character long alphanumeric string_          |
```
Comment (miguelaeh) [inline bitnami/harbor/README.md:330]:
```suggestion
| `jobservice.secret`               | Secret used when the job service communicates with other components. If a secret key is not specified, Helm will generate one. Must be a string of 16 chars. | Random 16 character long alphanumeric string            |
```

**Review by miguelaeh — APPROVED** [review]
Hi @walker-tom ,
Thank you very much for the PR!

> The comment above secretKey in values.yaml and in values-production.yaml gives not-a-secure-key as an example, are you happy for this to remain or would you like me to remove it?

Since it is an example we are happy with that comments.

LGTM!

**Review by miguelaeh — CHANGES_REQUESTED** [review]
Sorry I wanted to request the change in the README.md

**Comment by walker-tom** [comment]
All comments addressed 😃

**Review by miguelaeh — APPROVED** [review]
LGTM!

**Pull request merged**